libs/design/src/molecules/media-gallery/media-gallery.component.ts

```diff
@@ -19,11 +19,7 @@ export class DaffMediaGalleryComponent {
 	/**
 	 * The name of the gallery
 	 */
-	@Input('name') _name = '';
-
-	get name() {
-		return this._name ? this._name : uniqueGalleryId.toString();
-	}
+	@Input() name = uniqueGalleryId;
```

Comment:
This was causing a bug where all media gallery components were getting the same name, so clicking on any thumbnail would update the "selected image" of every media gallery